app/javascript/vue/tasks/dwca_import/components/settings/RequireTypeMaterialSuccess.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="label-above">
+    <label>
+      <input
+        type="checkbox"
+        v-model="requireTypeMaterialSuccess"
+      >
+      Skip/error rows where type material creation fails
```

Comment:
Not sure `Skip/error` will be well understood by users? I guess "Skip" here means maintain original behavior of proceeding with import regardless of type material import success?

What about "Prevent importing records with unprocessable `typeStatus` information"?

Reply:
Prevent sounds to me like a preemptive operation, like we're checking beforehand and we'll mark as Not Ready if it's unprocessable. Maybe "Error records with unprocessable `typeStatus` information"? 

It's challenging to phrase it precisely because "fail" is used for records that cause an internal error, rather than a data error.

Alternatively, what if it was an opt-out instead of opt-in? Something like 

"Ignore `typeStatus` information if unprocessable during record creation"

Reply:
To not alter current behavior just in case lets go with "Error records with unprocessable typeStatus information"

Reply:
Works for me.